src/app/register/register.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="registerForm">
+  <h1>Registration</h1>
+  <div  class="formField">
+    <input type="text" name="username" [(ngModel)]="username" placeholder="Username" required minlength="1">
+    <label for="username" class="error Message" *ngIf="usernameError">{{usernameError}}</label>
+  </div>
+  <div  class="formField">
+    <input type="password" name="password" [(ngModel)]="password" placeholder="Password" required minlength="8">
+    <label for="password" class="error Message" *ngIf="passwordError">{{passwordError}}</label>
+  </div>
+  <div  class="formField">
+    <input type="password" name="confirmPsw" [(ngModel)]="confirmPsw" placeholder="Confirm password" required minlength="8">
+    <label for="confirmPsw" class="error Message" *ngIf="confirmPswError">{{confirmPswError}}</label>
+  </div>
+  <div class="interactionsCont">
+    <button (click)="cancelRegistration()" class="cancelButton">Cancel</button>
+    <button (click)="registerUser()">Register</button>
+  </div>
+  <div class="bottomMessage">
+    <p class="success Message" *ngIf="messageSuccess">{{messageSuccess}}</p>
+    <p class="error Message" *ngIf="messageFail">{{messageFail}}</p>
+    <!-- <p class="success Message">Your account has been created.</p> -->
```

Comment:
Ez komment asszem :D valszeg ezt is kikéne venni vagy használni.

Reply:
köszi, kivettem